src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/CaseTakenOfflineRespondentNotificationHandler.java

```diff
@@ -50,12 +53,14 @@ public String camundaActivityId(CallbackParams callbackParams) {
     private CallbackResponse notifyRespondentSolicitorForCaseTakenOffline(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
 
-        notificationService.sendMail(
-            caseData.getRespondentSolicitor1EmailAddress(),
-            notificationsProperties.getSolicitorCaseTakenOffline(),
-            addProperties(caseData),
-            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
-        );
+        if (stateFlowEngine.hasTransitionedTo(callbackParams.getRequest().getCaseDetails(), CLAIM_NOTIFIED)) {
```

Comment:
this is not the right callback handler, `case proceeds in caseman` business process triggers `NOTIFY_RESPONDENT_SOLICITOR1_FOR_CASE_PROCEEDS_IN_CASEMAN` event

Reply:
🤦  two secs